source/test/meson.build

```diff
@@ -0,0 +1 @@
+#subdir('intltest')
```

Comment:
In comparison to 55.2 this is just dropped and commented out. Any particular reason?

Reply:
It did not compile due to reasons I could not decipher.